README.md

```diff
@@ -74,82 +136,65 @@ $ sudo td-agent-gem install fluent-plugin-geoip
   </record>
 
   # Settings for tag
-  remove_tag_prefix access.
-  tag               geoip.${tag}
+  tag               geoip.${tag[1]}
 
   # To avoid get stacktrace error with `[null, null]` array for elasticsearch.
   skip_adding_null_record  true
 
   # Set log_level for fluentd-v0.10.43 or earlier (default: warn)
   log_level         info
 
-  # Set buffering time (default: 0s)
-  flush_interval    1s
+  <buffer tag>
+    # Set buffering time (default: 0s)
+    flush_interval    1s
+  </buffer>
 </match>
 ```
 
 #### Tips: how to geolocate multiple key
 
 ```xml
 <match access.apache>
-  type geoip
+  @type geoip
   geoip_lookup_key  user1_host, user2_host
   <record>
     user1_city      ${city["user1_host"]}
     user2_city      ${city["user2_host"]}
   </record>
-  remove_tag_prefix access.
-  tag               geoip.${tag}
+  tag               geoip.${tag[1]}
 </match>
 ```
 
 #### Advanced config samples
 
 It is a sample to get friendly geo point recdords for elasticsearch with Yajl (JSON) parser.<br />
 
+**Notice** v0 config will be deprecated in the future.
 
 ```
 <match access.apache>
-  type                   geoip
+  @type                  geoip
   geoip_lookup_key       host
   <record>
     # lat lon as properties
     # ex. {"lat" => 37.4192008972168, "lon" => -122.05740356445312 }
-    location_properties  { "lat" : ${latitude["host"]}, "lon" : ${longitude["host"]} }
+    location_properties  '{ "lat" : ${latitude["host"]}, "lon" : ${longitude["host"]} }'
   
     # lat lon as string
     # ex. "37.4192008972168,-122.05740356445312"
     location_string      ${latitude["host"]},${longitude["host"]}
     
     # GeoJSON (lat lon as array) is useful for Kibana's bettermap.
     # ex. [-122.05740356445312, 37.4192008972168]
-    location_array       [${longitude["host"]},${latitude["host"]}]
+    location_array       '[${longitude["host"]},${latitude["host"]}]'
```

Comment:
Yes. This is v1 config. (Removed line is v0 config)
Fluentd v0.14 does not parse v0 config by default.
Fluentd v0.12 does not parse v0 config by default, too.
See `fluentd --help`.

v1 config can parse `location_array       '[${longitude["host"]},${latitude["host"]}]'`.
v0 config can parse both `location_array       '[${longitude["host"]},${latitude["host"]}]'` and `location_array       [${longitude["host"]},${latitude["host"]}]`.

BTW, v0 config does not support `@type` notation.
So we should use v1 config in README.md.